CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Fixed 
+- Mid rolls errorneously reported as post rolls when midroll's schedule time > the duration of the adbreak 
```

Comment:
I think it's about the ad duration, not ad break duration, right?

```suggestion
- Mid-roll ads erroneously reported as post-rolls when mid-roll's schedule time is greater than the duration of the ad
```

Reply:
Updated https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/118/commits/19815ff3a9dd1b44258aa68ef0c0c194044e6738